contracts/anchor_airdrop_registry/Cargo.toml

```diff
@@ -28,24 +28,21 @@ overflow-checks = true
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
-default = ["cranelift"]
 # for quicker tests, cargo test --lib
 # for more explicit tests, cargo test --features=backtraces
-backtraces = ["cosmwasm-std/backtraces", "cosmwasm-vm/backtraces"]
-cranelift = ["cosmwasm-vm/default-cranelift"]
-singlepass = ["cosmwasm-vm/default-singlepass"]
```

Comment:
Any need for these features @MSNTCS? We deleted them on all other contract upgrades for col5

Reply:
@ezaanm you can remove these we do not need them anymore.